modules/release-notes/ref-release-notes-developer-preview.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: REFERENCE
+[id="developer-preview"]
+= Developer Preview
+
+This section lists Developer Preview features in {product} {product-version}.
+
+[IMPORTANT]
+====
+Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to functionality in advance of possible inclusion in a Red Hat product offering. Customers can use these features to test functionality and provide feedback during the development process. Developer Preview features might not have any documentation, are subject to change or removal at any time, and have received limited testing. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+[id="developer-preview-rhdhpai-510"]
+== {product} Local
+
+{product-local} ({product-local-very-short}) is now available as a Developer Preview feature, providing a lightweight, self-contained version of {product-very-short} that allows developers and platform engineers to work on templates, try out plugins, validate software catalogs, and do other tasks without having to install {product-short} on a Kubernetes cluster.
+
+For more information about installing {product-local-very-short}, see link:https://github.com/redhat-developer/rhdh-local[{product-local} on Github].
+
```

Comment:
"Documentation is not included as part of the product’s documentation set." -- https://access.redhat.com/articles/6966848

But I guess this is pretty lightweight, even if not required.